reloj-checador/db/seeds/development.rb

```diff
@@ -1 +1,3 @@
-User.create({ name: 'george', email: 'geo@gmail.com', password: '123456' })
+Company.create([{ name: 'bcoders', address: 'c 34 #850 x 90 y 92' }, { name: 'wise', address: 'c 41 #652 x 70 y 72' },
+{ name: 'line', address: 'c 28 #356 x 21 y 23' }])
```

Comment:
keep the indentation, next object must be aligned to the first element (bcoders)